templates/vue/src/components/Lightbox/media/AnswerMedia.vue

```diff
@@ -0,0 +1,109 @@
+<template>
+  <div class="answers">
+    <h3>{{ node.title }}</h3>
+    <div
+      v-if="answer !== null"
+      class="answer-container mx-auto mb-3"
+      data-qa="answer-display"
+    >
+      <h2>{{ question.text }}</h2>
+      <h4 v-show="followUpText && hasAnswer" class="mb-4">
+        {{ answer.followUpText }}
+      </h4>
+      <h4 v-show="!followUpText && hasAnswer" class="mb-4">
+        {{ question.followUpText }}
+      </h4>
+      <b-tabs>
+        <b-tab v-for="questionAnswer in answers" :key="questionAnswer.type">
+          <template #title>
+            <div class="icon">
+              <tapestry-icon :icon="questionAnswer[0]" />
+              {{ questionAnswer[0] }}
+            </div>
+          </template>
+          <tapestry-activity
+            v-if="questionAnswer[0] === 'audio'"
+            :type="questionAnswer[0]"
+            :answerData="questionAnswer[1].url"
+          ></tapestry-activity>
+          <tapestry-activity
+            v-else
+            :type="questionAnswer[0]"
+            :answerData="questionAnswer[1]"
+          ></tapestry-activity>
+        </b-tab>
+      </b-tabs>
+      <div v-show="!hasAnswer" class="media-wrapper">
+        You have not completed this question yet.
+      </div>
+    </div>
+    <div v-else>
+      <p>Please fill in the answer form.</p>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapGetters, mapState } from "vuex"
+import TapestryActivity from "./ActivityMedia/TapestryActivity"
+import TapestryIcon from "@/components/common/TapestryIcon"
+export default {
+  name: "answer-media",
+  components: {
+    TapestryActivity,
+    TapestryIcon,
+  },
+  props: {
+    node: {
+      type: Object,
+      required: true,
+    },
+  },
+  computed: {
+    ...mapState(["userAnswers"]),
+    ...mapGetters(["getEntry", "getQuestion", "getAnswers"]),
+    answer() {
+      return this.node.answers
+    },
+    question() {
+      return this.getQuestion(this.answer.questionID)
+    },
+    followUpText() {

```

Comment:
There is already a followUp structure under each question. FollowUpText is not relevant anymore and should be referred to as question.followUp.text

Reply:
I think this line of code is correct, as the AnswerForm allows the user to override the question.followUp.text with their own followup text. If left blank, then it will default  to question.followUp.text. I changed the code in the template to reflect this.